bridge/server/app.ts

```diff
@@ -215,15 +171,12 @@ async function setAuth(app: Express, oAuthEnabled: boolean): Promise<{ authType:
   return { authType, session };
 }
 
-function setupDefaultLookAndFeel(): void {
+function setupDefaultLookAndFeel(mode: EnvType, rootFolder: string): void {
   try {
     log.info('Installing default Look-and-Feel');
 
     const destDir = join(rootFolder, 'dist/assets/branding');
-    const srcDir = join(
-      rootFolder,
-      `${process.env.NODE_ENV === 'development' ? 'client' : 'dist'}/assets/default-branding`
-    );
+    const srcDir = join(rootFolder, `${mode === EnvType.DEV ? 'client' : 'dist'}/assets/default-branding`);
```

Comment:
In production mode, the env-type is set to DEV, right? Then we would use `client` as base-dir, is this correct?

Reply:
In production, it is set to `production`